ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/spanning-tree.j2

```diff
@@ -52,21 +52,34 @@ STP Root Super: **{{ spanning_tree.root_super }}**
 {%     endif %}
 
 ### Global Spanning-Tree Settings
```

Comment:
Singe you are changing the documentation anyway, why not make this heading disappear if none of the knobs below are set.

Reply:
ok

Reply:
Done!